recipe/meta.yaml

```diff
@@ -54,7 +56,10 @@ test:
     - typing-extensions >=3.7.4  # [py<310]
   commands:
     - pip check
-    - pytest tests -vv -k "not (test_contents_unstaged or test_wheel.py or test_pep518 or test_tag or test_detect_wheel_tag or test_spam)" # tests are relevant to wheel and not conda packages
+    # CPPFLAGS exports -DNDEBUG which is unwanted
```

Comment:
The other way around. This removes `-DNDEBUG` from `CPPFLAGS` that is populated by our compiler activation scripts. Though, from what I saw, it doesn't have any effect...

Reply:
Sorry, that's me going too fast. Anyway, like I said, `-DNDEBUG` doesn't seem to cause issues.

Reply:
@danpetry because of this test: https://github.com/mesonbuild/meson-python/blob/0.15.0/tests/test_options.py#L37 which verifies that meson-python sets it correctly based on debug or release options being used. But since the compiler activation was presetting it, this was breaking the debug test.